test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -647,6 +651,78 @@ void ConfigureTestServices(IServiceCollection services)
         Assert.Equal("api/Values/{id}", aspnetcoreframeworkactivity.DisplayName);
     }
 
+    [Theory]
+    [InlineData("CONNECT")]
```

Comment:
A better way to test this would be to not rely on `RequestMethodHelper.KnownMethods` in this test and hardcode the expected string in `InlineData` itself.

Reply:
done